src/app/content/hooks/receiveContent.ts

```diff
@@ -23,6 +24,7 @@ const hookBody: ActionHookBody<typeof receivePage | typeof receiveBook> = ({getS
   dispatch(setHead({
     meta: [
       {property: 'og:description', content: ''},
+      {name: 'theme-color', content: theme.color.primary[book.theme].base},
```

Comment:
It seems like the theme color is being added twice. Is that desired?

![image](https://user-images.githubusercontent.com/253202/55431314-cf4caa80-5555-11e9-9cc7-7d2d4e3d767a.png)

Reply:
i have a note to myself that setHead is being sent twice on navigation and that is causing some tags to be added twice (og:description as well)

i will deal with that as a separate change